### ISSUE TITLE
[P1] Refuse completion-carrying TCQ units before packing the span-2 kernel lane

### ISSUE THREAD

**Issue by RobTand** [RobTand/tessera#296]
At `4e9744f6424ecd0d90ebcbd4200c909123f38a77`, `lane_planes.pack_unit_for_kernel` accepts a TCQ span-2 LUT unit with nonzero completion depth. Its checks cover transforms, start state, span, plane kind and rates (:501-527), but not the COMPLETION plane. The packed result (:534-544) omits both `completion_bits` and `completion_limit`; `build_subset_values` takes values through `build_anchor_values`, which always uses `forest.blocks[*][0]` (:193). The downstream `kernel.gemv_from_packed` forwards only these fields to `tessera_gemv_tuple_span2` (:1330-1334), so completion-selected descendants cannot reach the calculation.

This produces a different quantized matrix from `reconstruct_unit` for an ordinary supported encoder configuration. Scoped CPU reproduction (`PYTHONDONTWRITEBYTECODE=1 PYTHONPATH=src OMP_NUM_THREADS=1 TESSERA_FUSED_REPLAY=0 python3`):
```python
import torch
from dataclasses import replace
from tessera.alphabet import build_forest, E2M1_GRID, tuple_grid
from tessera.encode import encode_unit
from tessera.decode import reconstruct_unit
from tessera.manifest import ScalePlaneKind
from tessera.trellis import ConvCode
from tessera.lane_planes import pack_unit_for_kernel
f = build_forest(3, grid=tuple_grid(E2M1_GRID, 2))
c = ConvCode()
w = torch.randn((32,32), generator=torch.Generator().manual_seed(1)) * .02
u = encode_unit(w, f, (3,)*32, c, completion=1, span=2,
                scale_plane=ScalePlaneKind.LUT, scale_refit=0)
p = pack_unit_for_kernel(u, f, c)
z = replace(u, completion_bits=torch.zeros_like(u.completion_bits), completion_limit=0)
pz = pack_unit_for_kernel(z, f, c)
print(int((u.completion_bits != 0).sum()))
print(all(torch.equal(v,pz[k]) if isinstance(v,torch.Tensor) else v == pz[k]
          for k,v in p.items()))
a, b = reconstruct_unit(u,f,c), reconstruct_unit(z,f,c)
print(int((a != b).sum()), float((a-b).abs().max()))
```
Observed with torch 2.10.0+cpu: `319`, `True`, then `319 0.02685546875`. The full packed data are identical for two reconstructions that differ in 319 weights. This proves information loss before any kernel executes; no GPU GEMV/served KL result is claimed.

Expected: either pack and implement completion decoding, or refuse nonzero written completion depth at the shared packer. A named refusal is sufficient until that plane has a kernel implementation. Cover an ordinary completion=1/span=2 encode with nonzero words, alongside completion=0 and full-rate controls.

New finding adjacent to closed #246/PR266: those fixes correctly reject release, diagonals and rotation, but completion remains omitted. `tests/test_kernel.py`'s span-2 numerical fixtures explicitly use `completion=0` (:489, :664), and its legacy scalar completion refusal (:243) tests another wrapper. Existing receipts are accepted and do not cover this combination. Current shipping recipe defaults use completion=0, so this finding is scoped to the callable research/explicit kernel-lane path, not an assertion that a historical default serve was wrong.

The user requested issues only. No code was changed. The local refusal is implementable without a GPU; a future completion implementation requires independent kernel/reference evidence.

Disposition: filed during the requested review of closed audit issues and current code. No implementation changes were made; fixes are left for a separate dispatch.
